src/ImageRunModal.jsx

```diff
@@ -1101,21 +1178,37 @@
                            Dialogs.close();
                        }
                    }}
-                   title={this.props.pod ? cockpit.format(_("Create container in $0"), this.props.pod.Name) : _("Create container")}
-                   footer={<>
-                       <Button variant='primary' id="create-image-create-run-btn" onClick={() => this.onCreateClicked(true)} isDisabled={!image && selectedImage === ""}>
-                           {_("Create and run")}
-                       </Button>
-                       <Button variant='secondary' id="create-image-create-btn" onClick={() => this.onCreateClicked(false)} isDisabled={!image && selectedImage === ""}>
-                           {_("Create")}
-                       </Button>
-                       <Button variant='link' className='btn-cancel' onClick={Dialogs.close}>
-                           {_("Cancel")}
-                       </Button>
-                   </>}
+                   title={this.props.pod ? cockpit.format(_("Create container in $0"), this.props.pod.Name) : (this.props.prefill ? _("Clone container") : _("Create container"))}
+                   footer={
+                       <ImageRunModalFooter
+                           image={image}
+                           selectedImage={selectedImage}
+                           close={Dialogs.close}
+                           prefill={this.props.prefill}
+                           onCreateClicked={this.onCreateClicked} />
+                   }
             >
                 {defaultBody}
             </Modal>
         );
     }
 }
+
+const ImageRunModalFooter = ({ image, selectedImage, prefill, close, onCreateClicked }) => {
+    const createRunText = prefill ? _("Clone and run") : _("Create and run");
+    const createText = prefill ? _("Clone") : _("Create");
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230428-082025-ac2abc63-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)